src/main/java/calculator/Number.java

```diff
@@ -0,0 +1,54 @@
+package calculator;
+
+import java.util.Objects;
+
+public class Number {
+    private static final int MIN = 0;
+    static final int DEFAULT = 0;
```

Comment:
아래의 코드로 개선해 보면 어떨까요? 불필요한 객체 생성을 피할 수 있어요.

```suggestion
    static final Number ZERO = new Number(0);
```